about.html

```diff
@@ -7,67 +7,100 @@
   <title>Sprint Challenge - About</title>
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+  <link rel="stylesheet" href="css/about.css">
 
 </head>
 
 <body>
+
+   
+    <div class="navigation">
+    
+         <img src="img/lambda-black.png" alt="strategy"> 
+    
+    
+            <nav>
+              <a href="http://127.0.0.1:5500/Sprint-Challenge--User-Interface/#">Home</a>
+              <a href="#">About</a>
+              <a href="#">Products</a>
+              <a href="#">Blog</a>
+              <a href="#">Contact</a>
+          </nav>
+      </div>  
+
+      <div class="jumbo">
+      <img src="img/jumbo-about.png" alt="strategy">
+    </div>
+
     <div class="container about-page">
      
-    About
+     <div class="about">
+    <h3>About</h3>
 
     Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+    </div> 
 
-
+    <div class="boxes">
+      <div>
     <img src="img/about-plan.png" alt="strategy">
     
-    Strategy
+    <h3>Strategy</h3>
     
     Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+    <button>Learn More</button>
 
-    Learn More
-
-
+  </div>
+    
 
+    <div>
     <img src="img/about-working.png" alt="strategy">
```

Comment:
So all of the alt tags are the same here. The alt is used by screen readers for accessibility as well as Search Engines when ranking your site(this determines how high up on the list they will recommend it when it matches the search criteria). It is displayed when the image fails to load, as well as for those users who are visually impaired it will be read to them. So it is important to apply applicable `alt` properties.